### PR TITLE
ISSUE-195: PHP 8 docker container updates

### LINKED ISSUE #195: PHP 8 docker container updates
# What?
The PHP intl extension (for citeproc-php support) has been added to docker images so docker-compose files need to pull those. Additionally, the dev container needs xdebug config updates and a new filename convention (see esmero/archipelago-docker-images#46) so they can be overridden by users as needed.

### PR DESCRIPTION
**Pull request by aksm**
Resolves #195

### CODE REVIEW

**Review by DiegoPino — APPROVED** [review]
I trust you! It works right? 😃

**Comment by aksm** [comment]
@DiegoPino Your trust is misplaced! I think I'm still missing something. Not working on Linux at the moment, even after matching up all the configuration settings one-by-one with a working deployment. Will circle back in the morning and make another PR.

File: xdebug/xdebug.ini [hunk 1]
```diff
@@ -1,9 +1,15 @@
-zend_extension=xdebug
+; The below xdebug settings are part of the esmero-php-debug container. You may override any of the settings or add additional ones (see https://xdebug.org/docs/all_settings) as needed.
+
+;xdebug.remote_handler=dbgp
+;xdebug.client_port=9999
+;xdebug.start_with_request=yes
+;xdebug.connect_timeout_ms=500
+;xdebug.mode=develop,debug
+;xdebug.start_with_request=trigger
```
Comment (DiegoPino) [inline xdebug/xdebug.ini:8]: This might be the issue? xdebug.start_with_request twice, last one with "trigger" ?

**Review by DiegoPino — COMMENTED** [review]
Check the .ini ... I think the defaults are not as default as apparent

**Pull request merged**